tests/testapp/models/core.py

```diff
@@ -50,7 +50,10 @@
 
```

Comment:
Suggestion: rather than amending the existing tests to include the new functionality, could we add new tests that cover new behaviour and its edge cases? E.g. what happens when a string doesn't resolve?